src/reducers/filter.js

```diff
@@ -0,0 +1,10 @@
+const filter = (state = '', action) => {
```

Comment:
`state = ''`? Isn't it `'All'`?

([Redux will call our reducer with an undefined state for the first time. This is our chance to return the initial state of our app](https://redux.js.org/basics/reducers#handling-actions:~:text=Redux%20will%20call%20our%20reducer%20with,the%20initial%20state%20of%20our%20app)).